witness/src/Dscp/Snowdrop/Expanders.hs

```diff
@@ -139,7 +139,7 @@ seqExpandersBalanceTx =
         let inpNewBal     = aBalance inpPrevAcc + inputBack - inputSent
         let newInpAccount = Account
                 { aBalance = inpNewBal
-                , aNonce   = tiaNonce (txInAcc twTx) + 1
+                , aNonce   = tiaNonce (txInAcc twTx)
```

Comment:
Wait, what?

Reply:
The nonce was incremented in two places and caused off-by-one errors.
Probably, this wasn't the best place to fix it, I'll try making it a bit more clear.

Reply:
By using more descriptive names, like, lastNonce and nextNonce, among other things :)

Reply:
Oh, actually this was the right place to change it.
This one here is the new account _after_ tx application.
Accounts store last used nonce, so there was no sense in incrementing it here.